common/src/main/java/com/wynntils/core/functions/FunctionManager.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.functions;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.functions.TestFunction;
+import com.wynntils.functions.WorldNameFunction;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.TextComponent;
+
+/** Manage all built-in {@link Function}s */
+public final class FunctionManager {
```

Comment:
is this not a manager? should we call this registry?

Reply:
Ah, that's what you meant. Hm. I started by calling it FunctionRegistry, but then it got more logic like `getSimpleValueString`, or the rather complex `enableFunction` so I renamed it "manager" to reflect that it was just not a simple and stupid registry.

However, it does not listen to subscription events. And the functions themselves have I18n support. ... but that's not needed at class creation time... right. Yeah, ok, fine, let's make this a CoreManager then.